ext/MitsubaLoader/CMitsubaLoader.h

```diff
@@ -309,126 +38,30 @@ class CMitsubaLoader : public asset::IAssetLoader
 		struct SInstanceData
 		{
 			core::matrix3x4SIMD tform;//mat4x3
+			//elements (0,3) and (1,3) are (first,count) for bsdf_instrStream (remainder and pdf stream in case of RT backend)
 			core::matrix3x4SIMD normalMatrix;//mat4x3
+			std::pair<uint32_t,uint32_t> prefetch_instrStream;
+			std::pair<uint32_t, uint32_t> nprecomp_instrStream;
+			std::pair<uint32_t, uint32_t> genchoice_instrStream;
```

Comment:
wouldn't it make more sense to store just one offset and 3 counts?

Reply:
why not

Reply:
is this a "why not" in agreement?

Reply:
yeah, all streams for one IR tree are put in continuous memory anyway so it can be done without an issue